doc/man7/openssl-quic.pod

```diff
@@ -498,6 +504,57 @@ The following SSL APIs are new but relevant to both QUIC and DTLS:
 
 =over 4
 
+=item L<SSL_new_listener(3)>
+
+Creates a listener SSL object. A listener SSL object differs from an ordinary
+SSL object in that it is used to provide an abstraction for the acceptance of
+network connections in a protocol-agnostic manner.
+
+Currently, listener SSL objects are only supported for QUIC server usage or
+client-only usage. It is expected that the listener interface will be expanded
+to support other protocols.
+
+=item L<SSL_new_listener_from(3)>
+
+Creates a listener SSL object which is subordinate to a QUIC domain SSL object
+I<ssl>. See L<SSL_new_domain(3)> and L<openssl-quic-concurrency(7)> for details
+on QUIC domain SSL objects.
+
+=item L<SSL_is_listener(3)>
+
+Returns 1 if and only if a SSL object is a listener SSL object.
+
+=item L<SSL_get0_listener(3)>
+
+Returns an SSL object pointer (potentially to the same object on which it is
+called) or NULL.
+
+=item L<SSL_listen(3)>
+
+Begin listening after a listener has been created. It is ordinarily not needed
+to call this because it will be called automatically on the first call to
+L<SSL_accept_connection(3)>. 
+
+=item L<SSL_accept_connection(3)>
+
+Accepts a new incoming connection for a listner SSL object. A new SSL object
+representing the accepted connection is created and returned on success. If no
+incoming connection is available and the listener SSL object is configured in
+nonblocking mode, NULL is returned.
+
+=item L<SSL_get_accept_connection_queue_len(3)>
+
+Returns an informational value listing the number of connections waiting to be
+popped from the queue via calls to SSL_accept_connection().
+
+=item L<SSL_new_from_listener(3)>
```

Comment:
Note that this function appears to be currently unimplemented (it always returns NULL). I hope there is an issue somewhere to implement it...!!

Reply:
I couldnt find an issue tracking this, so I've opened one to track it: https://github.com/openssl/openssl/issues/25851